BigQuery.Linq.Tests/Builder/StandardQueries.cs

```diff
@@ -505,13 +505,6 @@ public void EnumAsValue()
         [TestMethod]
         public void DateTime()
         {
-            var jpCalendar = CultureInfo.GetCultureInfo("ja-jp").Calendar; // +9
-            new BigQueryContext().Select(() => new { dt = new DateTime(2014, 10, 17, 6, 0, 0, jpCalendar) })
-                .ToString()
-                .Is(@"
-SELECT
-  '2014-10-16 21:00:00.000000' AS [dt]".TrimSmart());
-
```

Comment:
Sorry, I undesrstand.
Is this OK?
I want to test auto convert localtime to utc.

```csharp
var offset = TimeZone.CurrentTimeZone.GetUtcOffset(System.DateTime.Now);
var local = new DateTime(new DateTime(2014, 10, 16, 21, 0, 0, DateTimeKind.Utc).Ticks + offset.Ticks, DateTimeKind.Local);
new BigQueryContext().Select(() => new { dt = local })
                .ToString()
                .Is(@"
SELECT
  '2014-10-16 21:00:00.000000' AS [dt]".TrimSmart());
```